LogicAppsCustomCodeFramework/WorkflowLogicApp/FlowToInvokeFunction/workflow.json

```diff
@@ -0,0 +1,41 @@
+{
+    "definition": {
+        "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
+        "actions": {
+            "Invoke_a_function_in_this_Logic_App": {
+                "type": "InvokeFunction",
+                "inputs": {
+                    "functionName": "FlowInvokedFunction",
+                    "parameters": {
```

Comment:
are there any instructions someone who wants to use this project will need? If so, I suggest adding a readme file at the top of the folder structure.